src/mumble/mumble_pch.hpp

```diff
@@ -81,6 +81,9 @@
 #define interface struct
 #endif
 
+#ifndef _WIN32_WINNT
```

Comment:
The problem is:
```
/usr/lib/mxe/usr/i686-w64-mingw32.static/include/qos2.h:87:3: error: 'PQOS_FLOWID' has not been declared
   PQOS_FLOWID FlowId
   ^
/usr/lib/mxe/usr/i686-w64-mingw32.static/include/qos2.h:112:3: error: 'QOS_FLOWID' has not been declared
   QOS_FLOWID FlowId,
   ^
/usr/lib/mxe/usr/i686-w64-mingw32.static/include/qos2.h:122:3: error: 'QOS_FLOWID' has not been declared
   QOS_FLOWID FlowId,
   ^
/usr/lib/mxe/usr/i686-w64-mingw32.static/include/qos2.h:133:3: error: 'QOS_FLOWID' has not been declared
   QOS_FLOWID FlowId,
   ^
```

_typedef UINT32 QOS_FLOWID_ is only in _qos2.h_, but at line 141, after the things that require it...